pairtaghighlighter/src/pair_tag_highlighter.c

```diff
@@ -319,6 +321,9 @@ static void findMatchingTag(ScintillaObject *sci, gint openingBracket, gint clos
     gboolean isTagOpening = is_tag_opening(sci, openingBracket);
     gchar *tagName = get_tag_name(sci, openingBracket, closingBracket, isTagOpening);
 
+    if (!tagName)
```

Comment:
> And check on line 369 preserves from it (marked with comment "Don't run search on empty brackets").

Oh.  It's not sufficient with something like `< href="foo">` as the brackets won't technically be empty (so the check you're talking about will pass), but `get_tag_name()` will return NULL.  And boom might happen (`is_tag_empty()` receiving `NULL`, or `strcmp()` alike -- but `strcmp()` doesn't have the debugging safeguard).